wcivf/assets/js/scripts.js

```diff
@@ -1,28 +1,28 @@
-fallback.ready(function() {
-    var feedback_form = document.forms[1]
-    var feedback_form_token = feedback_form.elements[1].value
-    var csrfmiddlewaretoken = feedback_form.elements[0].value
-    document.querySelector('[name=found_useful]').click(function(el) {
-        var found_useful = document.querySelector(this).value;
-        fetch('/feedback/submit_initial/', {
-            method: "POST",
-            body: JSON.stringify(data),
-            found_useful: found_useful,
-            token: feedback_form_token,
-            csrfmiddlewaretoken: csrfmiddlewaretoken,
-            source_url: window.location.pathname
-        }).then(res => {
-            console.log(body);
-        });
+feedback.ready(function() {
```

Comment:
Fallback is actually not a typo here:

https://github.com/dolox/fallback/

We used to use this library, but it looks like it's not installed any more. This change cases an error in the console for me, but so does `fallback`. I think we actually want to unwrap the call here, and just add the javascript directly. Or maybe we need to use `document.ready`?

Reply:
I see above you've added the JS inline in the HTML. I don't think that'll be needed if you fix this code